packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -42,9 +42,9 @@ export interface ButtonGroupProps
 }
 
 const alignmentMap = {
-  left: 'flex-end',
+  left: 'flex-start',
   center: 'center',
-  right: 'flex-start',
+  right: 'flex-end',
```

Comment:
Since we added the `row-reverse` we need to inverse those to keep the correct alignent.

Reply:
🤦 you're right, thanks